src/test/java/uk/gov/ons/ctp/integration/rhsvc/repository/RespondentCaseRepositoryIT.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.ons.ctp.integration.rhsvc.repository;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.ons.ctp.common.FixtureHelper;
+import uk.gov.ons.ctp.common.event.model.CaseUpdate;
+import uk.gov.ons.ctp.integration.rhsvc.FirestoreTestBase;
+
+public class RespondentCaseRepositoryIT extends FirestoreTestBase {
+  private static final String CASE_ID = "dc4477d1-dd3f-4c69-b181-7ff725dc9fa4";
+
+  @Autowired private RespondentCaseRepository caseRepo;
+
+  @BeforeEach
+  public void setup() throws Exception {
+    deleteAllCollections();
+  }
+
```

Comment:
this would be a great place to add a real firestore test for the `readCaseUpdateBySampleAttribute` method.

Reply:
New test added.